.github/workflows/release-apworld.yml

```diff
@@ -76,5 +81,6 @@ jobs:
           artifacts: '*'
           allowUpdates: true
           prerelease: ${{ inputs.prerelease }}
+          immutableCreate: ${{ inputs.immutable-create }}
```

Comment:
I wonder if we could set that to true all the time. I think github might throw a fit but it's worth having a look

Reply:
Would it? Because AFAICT from the action the only thing this changes is that it creates the original release as draft and then publishes it. So I'm not sure it would fail for updates. I don't have time to test right now but I'll try to think about it tonight